apiml-common/src/main/resources/apiml-common-log-messages.yml

```diff
@@ -0,0 +1,39 @@
+messages:
+
+    # Info messages
+    # 000-100
```

Comment:
000-99 and so on, for all other categories

Reply:
adjusted